app/scripts/extensions/nav/dropdownMobile.js

```diff
@@ -24,38 +24,38 @@ angular.module('openshiftConsole')
         dropdownItems = dropdownItems.concat([{
           type: 'dom',
           node: [
-            '<li>',
+            '<li class="list-group-item">',
```

Comment:
This will break existing extensions if we require a new class on menu items.

@rhamilto @jeff-phillips-18 Any ideas on how we can avoid that?

Reply:
I have this fixed except for active styles.

Reply:
The active styles work now too